src/formState.ts

```diff
@@ -673,8 +673,10 @@ function newValueFieldState<T, K extends keyof T>(
         throw new Error(`${key} is currently readOnly`);
       }
 
-      if (opts.refreshing && this.dirty && value !== this.value) {
-        // Ignore refreshed values if we're already dirty
+      if (opts.refreshing && this.dirty && this.value !== value) {
```

Comment:
I guess there is no actual change here, I just flipped `this.value` and `value` to match the order I happened to type them in in the comment.